src/VisualStudio.Common/GlobalSuppressionsOptionsPageControl.xaml.cs

```diff
@@ -20,39 +20,77 @@ public GlobalSuppressionsOptionsPageControl()
 
         private void OpenLocation_Click(object sender, RoutedEventArgs e)
         {
-            string assemblyPath = typeof(GlobalSuppressionsOptionsPageControl).Assembly.Location;
+            string appDataFolder = Path.Combine(
+                Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), 
+                @"JosefPihrt\Roslynator\VisualStudio\2019");
+            string ruleSetPath = Path.Combine(appDataFolder, "roslynator.ruleset");
 
-            if (!string.IsNullOrEmpty(assemblyPath))
+            if (!File.Exists(ruleSetPath))
             {
-                string assemblyDirPath = Path.GetDirectoryName(assemblyPath);
-
-                if (!string.IsNullOrEmpty(assemblyDirPath))
+                try
                 {
-                    string ruleSetPath = Path.Combine(assemblyDirPath, "roslynator.ruleset");
-
-                    try
+                    string defaultRulesetFileName = GetDefaultRulesetFileName();
+                    if (defaultRulesetFileName != null && File.Exists(defaultRulesetFileName))
                     {
-                        string fileToSelect = (File.Exists(ruleSetPath)) ? ruleSetPath : assemblyPath;
+                        if (!Directory.Exists(appDataFolder))
+                        {
+                            Directory.CreateDirectory(appDataFolder);
+                        }
+                        File.Copy(defaultRulesetFileName, ruleSetPath);
+                    }
+                }
+                catch (Exception ex)
+                {
+                    if (ex is InvalidOperationException
```

Comment:
These exception are not relevant, should be IOException and UnauthorizedAccessException

Reply:
Done